examples/chatgpt-apps/Tic_Tac_Soccer/src/GameEngine.ts

```
@@ -0,0 +1,1863 @@
+        return pos.includes('MF') || pos.includes('MIDFIELDER') || pos.includes('CM') || 
+               pos.includes('DM') || pos.includes('AM') || pos.includes('LM') || pos.includes('RM');
+      
+      case 'FORWARD':
+        // Check for any forward position indicators
+        return pos.includes('FW') || pos.includes('FORWARD') || pos.includes('ST') || 
+               pos.includes('SS') || pos.includes('LW') || pos.includes('RW') || pos.includes('CF') ||
+               pos.includes('WING') || pos.includes('ATTACK');
+      
+      default:
+        // Fallback: normalize and compare (handles case variations)
+        const normalized = this.normalizePosition(player.Position);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)